core/escrow.js

```diff
@@ -37,58 +72,68 @@ const escrowExchange = {
         getRefund() {
           return refund.p;
         },
+        getUse() {
+          return E(escrowP).getUse();
+        },
       });
     }
 
     // Promise wiring
 
-    const moneyPayment = makePromise();
-    const moneyTransfer = makeTransfer(moneyNeeded, moneyPayment.p);
+    const leftOfferPayment = makePromise();
+    const leftEscrow = makeTransfer(leftOfferAmount, leftOfferPayment.p);
 
-    const stockPayment = makePromise();
-    const stockTransfer = makeTransfer(stockNeeded, stockPayment.p);
+    const rightOfferPayment = makePromise();
+    const rightEscrow = makeTransfer(rightOfferAmount, rightOfferPayment.p);
 
     // TODO Use cancellation tokens instead.
-    const aliceCancel = makePromise();
-    const bobCancel = makePromise();
+    const leftCancel = makePromise();
+    const rightCancel = makePromise();
 
     // Set it all in motion optimistically.
 
     const decisionP = Promise.race([
-      Promise.all([moneyTransfer.phase1(), stockTransfer.phase1()]),
-      aliceCancel.p,
-      bobCancel.p,
+      Promise.all([leftEscrow.phase1(), rightEscrow.phase1()]),
+      leftCancel.p,
+      rightCancel.p,
     ]);
     decisionP.then(
       _ => {
-        moneyTransfer.phase2();
-        stockTransfer.phase2();
+        leftEscrow.phase2();
+        rightEscrow.phase2();
       },
       reason => {
-        moneyTransfer.abort(reason);
-        stockTransfer.abort(reason);
+        leftEscrow.abort(reason);
+        rightEscrow.abort(reason);
       },
     );
 
     // Seats
 
-    const aliceSeat = harden({
-      offer: moneyPayment.res,
-      cancel: aliceCancel.reject,
-      getWinnings: stockTransfer.getWinnings,
-      getRefund: moneyTransfer.getRefund,
+    const coreLeftSeat = harden({
+      offer: leftOfferPayment.res,
+      cancel: leftCancel.reject,
+      getWinnings: rightEscrow.getWinnings,
+      getRefund: leftEscrow.getRefund,
     });
 
-    const bobSeat = harden({
-      offer: stockPayment.res,
-      cancel: bobCancel.reject,
-      getWinnings: moneyTransfer.getWinnings,
-      getRefund: stockTransfer.getRefund,
+    const coreRightSeat = harden({
+      offer: rightOfferPayment.res,
+      cancel: rightCancel.reject,
+      getWinnings: leftEscrow.getWinnings,
+      getRefund: rightEscrow.getRefund,
     });
 
+    const leftSeat = makeCustomLeftSeat(coreLeftSeat, leftEscrow, rightEscrow);
```

Comment:
Sure, what were you thinking?

Reply:
The only doc currently is at the top of escrow.js. I don't know of a better place, so that would do.

I'd like to see comments that say that escrow supports customizable behavior for the seats, and then describe how the customization is provided, and what interface it can count on. I *think* the intent is that the customization can add new behaviors that have access to the respective coreSeat, and both deposits. 

I'm not sure what I would say about what kinds of interactions this is intended to support. Is the goal that they mainly add other ways to close out the escrow? I don't see enough here to know how else they might interact while the escrow is ongoing and not interfere with the core functionality.